index.html

```diff
@@ -73,7 +73,230 @@ <h1>
           <rect x="1" y="1" width="40" height="40" rx="20" stroke="#DFE1E6" />
         </svg>
       </div>
-    </div>
+    </section>
+    <section class="recent-works">
+      <div class="section-heading">
+        <h1>My Recent<br>Works</h1>
+        <div class="indicator"></div>
+      </div>
+      <div class="card-container">
+        <div class="card">
+          <div class="image"></div>
```

Comment:
- [ ] According to the design each project has an image. You can either set it as a background image for this div but I think it's better to use the img tag for it. Because the image is related to the content of the webpage.

Reply:
hi thanks the template I'm using has only a image placeholder should I use my own images?

Reply:
Yes I think you can use any image as a placeholder.